src/frontend/hooks/useAppLanguageTag.ts

```diff
@@ -0,0 +1,17 @@
+import {useLocales} from 'expo-localization';
+import {useMemo} from 'react';
+
+import {useLocaleState} from '../contexts/LocaleStoreContext';
+import {getAppLanguageTag} from '../lib/intl';
+
+export function useAppLanguageTag() {
```

Comment:
This currently returns an object that has this representation:

```ts
type ResolvedLanguageTag = {
  source: 'fallback' | 'system' | 'selected';
  value: SupportedLanguageTag;
};
```

The `source` field is not really used in the app, but I find it potentially helpful for debugging. Open to just returning the `value` field when it comes to this hook specifically though.